cpp/dolfinx/geometry/BoundingBoxTree.cpp

```diff
@@ -283,7 +280,7 @@ BoundingBoxTree::BoundingBoxTree(
   }
 
   LOG(INFO) << "Computed bounding box tree with " << num_bboxes()
-            << " nodes for " << num_leaves << " points.";
+            << " nodes for " << points.size() << " points.";
```

Comment:
This makes `num_leaves` above unused.

Reply:
It's used in the `if` statement just above.